contentcuration/search/serializers.py

```diff
@@ -1,18 +0,0 @@
-from contentcuration import models as cc_models
```

Comment:
I assume this just wasn't being used at all from what I can see?

Reply:
Yes - it was never used anywhere